Commands/Process.php

```diff
@@ -37,13 +35,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $systemCheck = new SystemCheck();
         $systemCheck->checkRedisIsInstalled();
 
-        Access::getInstance()->setSuperUserAccess(false);
-        Plugin\Manager::getInstance()->setTrackerPluginsNotToLoad(array('Provider'));
+        $trackerEnvironment = new Environment('tracker');
```

Comment:
Will this recreate the container and clear any previous created instances? Especially re the Cache? I'm not into the environment / container code so just asking :)